### ISSUE TITLE
Seconds as optional parameter

### ISSUE THREAD

**Issue by diogonborges** [quartznet/quartznet#1253]
**Is your feature request related to a problem? Please describe.**
Any chance that the cron expressions would be supporting seconds to be optional?

**Describe the solution you'd like**
Ideally like to have support for expressions like: * * * * *

**Describe alternatives you've considered**
I've considered other libs packages?! 
Sorry, I don't know how to best describe alternatives other than adopting the cron expression format

**Additional context**

**Comment by lahma** [comment]
So you would not want to have 0 as first parameter, that's all it takes to "ignore" the seconds part?

**Comment by diogonborges** [comment]
That's correct @lahma.
Is it a weird expectation? 
I've read through the documentation: https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/crontriggers.html#example-cron-expressions to come up with the awareness that **Year** is optional but **Seconds** not.

Is there a reason for this? My fairtly un-educated view on this would tend to think the think the most adopted flavour of cron expressions takes after Unix Style and would use 5 or 6 fields.

Thanks for reaching back!

**Comment by lahma** [comment]
Yes, for Unix-like systems second granularity wouldn't make sense because of the overhead involved in starting the process. Quartz.NET follows Java version's logic allowing to utilize cron for seconds-level accuracy for firing. So it's for convenience and unfortunately makes it different from what a systems admin would expect. If there would be a backwards compatible way to support both, why not.

**Comment by lahma** [comment]
Alpha.5 design ratified (alpha5-note-cron.md §6) — this ships as an **explicit format switch, not auto-detect**:

- `CronFormat` enum (`Quartz` = 0 default, `Unix`), surfaced on `Parse`/`TryParse` and `CronScheduleBuilder.Create` only — four baseline lines, one type. No constructor overload, no `WithCronSchedule` overload (compose via `WithCronSchedule(CronExpression.Parse(s, CronFormat.Unix))`), and no XML/HTTP/dashboard format input: a 5-field expression is something you write in C#, not something you store.
- Auto-detect was examined and rejected on this repository's own evidence: three separate test fixtures use "5 fields" to mean "invalid expression"; dropping a middle field would become silent (`0 0 12 * * ?` minus the month = midnight on the 12th instead of noon daily); the same digit means different days across dialects; and auto-detect can never grow to Spring's 6-field format. Adding auto-detect later is additive; removing it would be impossible — at a freeze, take the reversible option.
- The Unix rewrite is a pure string transform ahead of the existing parser: day-of-week renumbered `n → (n % 7) + 1` (0 and 7 both Sunday), whole-week ranges degenerate to `*`, `L`/`W`/`#`/`H` still work inside the 5-field layout, output normalized to the canonical Quartz 6-field form. **The store gains no format column**; `CronExpressionString` and the dashboard show the canonical form and the original text is deliberately not recoverable — the same trade as the existing uppercasing.
- **The `@` macros ship unconditionally and dialect-free** (`@yearly`/`@annually`/`@monthly`/`@weekly`/`@daily`/`@midnight`/`@hourly` — the Vixie set; no `@every_*`, no jitter variants; `@reboot` rejected by name with guidance). They work at every entry point including XML and the dashboard with zero new API, recovering most paste-from-crontab ergonomics even where the format switch is absent.
- Cleanly separable in three layers (macros / rewrite / surface), so the cut-to-ruling fallback stays available. Depends on #3589 — the rewriter leans on the restricted-field rule.
